### PR TITLE
Addition of module-level logging.

### PR DESCRIPTION
**Pull request by will-rice**
This PR adds logging to the library in similar locations as the android implementation.

### CODE REVIEW

File: spokestack/wakeword/tflite.py [hunk 1]
```diff
@@ -178,6 +184,7 @@ def _detect(self, context: SpeechContext) -> None:
 
         if posterior > self._posterior_threshold:
             context.is_active = True
+            _LOG.info(f"wake: {self._posterior_max}")
```
Comment (brentspell) [inline spokestack/wakeword/tflite.py:187]: we want to log the maximum posterior after we update it (I believe this was also wrong in android), so let's move lines 188-189 before 185

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
         "pyaudio",
         "webrtcvad",
         "numpy",
-        "websocket",
+        "websocket_client",
```
Comment (brentspell) [inline setup.py:27]: this should at least be a different commit, probably a separate pr

Comment (will-rice) [reply setup.py]: yeah this was an accident. we will make the change though because `websocket_client` is the one we use. there are a couple with similar names which is annoying.
